src/api/config/Config.ts

```diff
@@ -15,6 +15,7 @@ export default class Config implements StrykerOptions {
   timeoutFactor = 1.5;
   plugins: string[] = ['stryker-*'];
   port = 9234;
+  reporter = ['progress', 'clear-text'];
```

Comment:
Overridden. This is the way Karma works as well.. You want to give the people the opportunity to use something else entirely